source/guides/getting-started/writing-your-first-test.md

```diff
@@ -86,6 +86,8 @@ describe('My First Test', function() {
 
 Once you save again, you'll see Cypress display the failing test in red since `true` does not equal `false`.
 
+Cypress also displays the stack trace and the code frame where the assertion failed (when available). You can click on the blue file link to open the file where the error occurred in your preferred application. To read more about the error's display, read about {% url "Debugging Errors" debugging#Errors %}.
```

Comment:
Should we link "your preferred application" to the file opener preference section?

Reply:
Yes. updated 👍